src/plugin-styles.scss

```diff
@@ -27,12 +27,8 @@ $modal-footer-height: 70px;
 
 	&__plugin-sidebar {
 		&__about-button {
-			color: $gray-600;
+			color: $gray-700;
 			margin-top: 3rem;
 		}
-
-		&__about-button svg {
-			fill: $gray-600;
-		}
```

Comment:
The `svg` element in the button component is applied with `currentColor` by default.

https://github.com/WordPress/gutenberg/blob/4d1aa40f02c6a46e08b1496f74c66a059611b0a2/packages/components/src/button/style.scss#L345